Source/Core/DolphinQt2/GameList/GameListModel.cpp

```diff
@@ -25,8 +25,10 @@ GameListModel::GameListModel(QObject* parent) : QAbstractTableModel(parent)
   connect(&m_tracker, &GameTracker::GameRemoved, this, &GameListModel::RemoveGame);
   connect(&Settings::Instance(), &Settings::PathAdded, &m_tracker, &GameTracker::AddDirectory);
   connect(&Settings::Instance(), &Settings::PathRemoved, &m_tracker, &GameTracker::RemoveDirectory);
-  connect(&Settings::Instance(), &Settings::PathReloadRequested, &m_tracker,
-          &GameTracker::ReloadDirectory);
+  connect(&Settings::Instance(), &Settings::PathReloadRequested, this, [this](const QString& path) {
+    m_title_database = Core::TitleDatabase();
```

Comment:
This reloads the title database once for each directory that gets scanned. It works, but it's a bit unnecessary.

Reply:
Should only happen once now.